docs/setupstoragevolumes.md

```diff
@@ -8,81 +8,228 @@ sidebar_label: Storage Volumes
 
 ### Storage Volumes
 
-A Storage Class provides a way for administrators to describe the “classes” of storage they offer. Different classes may map to quality-of-service levels, backup policies, or to arbitrary policies determined by the cluster administrators. This concept is sometimes called “profiles” in other storage systems.
+A OpenEBS Volume comprises of a Target and Replica component. There can be one or more Replicas. The Replica component are the ones that access the underlying disk resources for storing the data. Storage volumes need to be persistent to the application.
```

Comment:
An OpenEBS
The Replica component accesses the ...
Storage volumes must be persistent with the application.